book/src/08_parallel_join.md

```diff
@@ -45,8 +47,10 @@ fn run(&mut self, (vel, mut pos): Self::SystemData) {
 }
 ```
 
-The `par_join` method produces a type implementing [`rayon`s `ParallelIterator`][ra]
+There is always overhead in parallelization, so you should always carefully profile to see if there are benefits in the switch. If you have only few things to iterate over then sequential join is faster.
```

Comment:
Please remove the second "always". I'd probably put this into a little "box" by placing "\>" before the text.